### PR TITLE
Bah 18658

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/18658

## Screenshots
Screen Shot 2019-05-30 at 1 49 35 PM
Screen Shot 2019-05-30 at 1 50 06 PM
Screen Shot 2019-05-30 at 1 50 14 PM
Screen Shot 2019-05-30 at 1 50 14 PM
Screen Shot 2019-05-30 at 1 50 23 PM
Screen Shot 2019-05-30 at 1 50 33 PM

## Acceptance criteria
- [ ] The "STEM" questions page is displayed when the user selects one of the following answers on the first "Education Benefit" page:
     - "Post-9/11 GI Bill (Chapter 33)"
     - "Fry Scholarship (Chapter 33)"
- [ ] The "STEM" questions page has the following questions:
     - "Are you applying for the Edith Nourse Rogers STEM Scholarship (Chapter 33)? Learn more" 
     - "Are you enrolled in an undergraduate STEM degree program?"
     - "Do you have a STEM undergraduate degree and are now pursuing a teaching certification?"
- [ ] The available answers to all questions in AC-2 are "Yes" and "No".
- [ ] The question "Are you applying for the Edith Nourse Rogers STEM Scholarship (Chapter 33)? Learn more" has the following behaviors:
     - If the answer is "Yes" it conditionally displays the "Are you enrolled in an undergraduate STEM degree program?" question.
     - If the answer to the question is "No"; the workflow continues to the first page of the "Military History" section.
     - "Learn more" is a link that opens a modal with placeholder text (lorem ipsum).
- [ ] The question "Are you enrolled in an undergraduate STEM degree program?" has the following behaviors:
     - If the answer is "Yes"; the workflow continues to the first page of the "Military History" section.
     - If the answer is "No" the question "Do you have a STEM undergraduate degree and are now pursuing a teaching certification?" is displayed.
- [ ] The question "Do you have a STEM undergraduate degree and are now pursuing a teaching certification?" has the following behaviors:
     - If the answer is "Yes"; the workflow continues to the first page of the "Military History" section.
     - If the answer is "No"; the workflow continues to the first page of the "Military History" section.

       

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/1995/content/stemInformation.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```
Comment (zurbergram) [inline src/applications/edu-benefits/1995/content/stemInformation.jsx]: file name should be the same as the page name

**Review by zurbergram — APPROVED** [review]
LGTM

File: src/applications/edu-benefits/1995/config/new-chapters.js [hunk 2]
```diff
@@ -66,6 +66,16 @@ export const newChapters = {
           },
         },
       },
+      // related to 1995-STEM
+      stem: {
+        title: 'Education benefit',
+        path: 'benefits/stem',
+        depends: {
```
Comment (zurbergram) [inline src/applications/edu-benefits/1995/config/new-chapters.js]: this should be a function

**Comment by hughes-dustin** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Review by rianfowler — APPROVED** [review]

**Pull request merged**